### DISCUSSION TITLE
Sporadisch merkwürdige Werte seit ein paar Wochen beim Wasserzähler

### DISCUSSION THREAD

**Discussion by AdmiralRaccoon** [jomjol/AI-on-the-edge-device#2474] [General]
### The Problem

Moin,
ich hab Gas & Wasserzähler seit Oktober laufen, ohne etwas an der config zu ändern Der Wasserzähler spinnt allerdings seit ein paar Wochen sporadisch und liest einen Wert aus, wo ich mich Frage, wie er darauf annähernd kommen kann?! (Siehe Bild)

Ich setze dann immer den neuen Wert als "previous value" manuell.

Fairerweise muss ich sagen, das die beiden ESP´s seit bestimmt Dezember/Januar ununterbrochen am Strom hängen, ob der deshalb manchmal zicken macht?

### Version

15.2.4

### Logfile

```shell
[3d19h01m11s] 2023-06-13T06:34:47	<ERR>	[POSTPROC] main: Raw: 00138.3183, Value: , Status: Neg. Rate - Read:  - Raw: 00138.3183 - Pre: 11111.7496 
[3d20h01m11s] 2023-06-13T07:34:47	<ERR>	[POSTPROC] main: Raw: 00138.3357, Value: , Status: Neg. Rate - Read:  - Raw: 00138.3357 - Pre: 11111.7496
```

### Screenshots

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/41727850/b0dfe5bd-d281-4065-8d75-e2c9809242a8)

**Comment by ghost** [comment]
Das kommt, wenn das Alignment nicht richtig funktioniert. Hier hilft nur daran weiter zu arbeiten.

**Comment by caco3** [comment]
Das alignment wird genauer, wenn du dazu möglichst weit von einander entfernte Punkte verwendest.

Beispiel:
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/1783586/6c433a13-0469-426d-80d9-99cd4e221f57)

**Reply by AdmiralRaccoon** [reply]
Also in meinem Fall eher das "Ernst" und die "m³" ?

**Reply by caco3** [reply]
ja

**Comment by caco3** [comment]
Die Laufzeit sollte kein Problem sein.

Wieso hast Du die Pre-Value deaktiviert?

**Reply by AdmiralRaccoon** [reply]
Guten Frage...ich glaub ich hatte da am Anfang Stress mit und seitdem war das deaktiviert. 
Tatsächlich hatte ich jetzt auch seit einer Woche diese krasse Fehllesung nicht mehr, keine Ahnung warum der das ab und zu mal hatte.
